cmd/nodepool/create.go

```diff
@@ -0,0 +1,128 @@
+package nodepool
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/spf13/cobra"
+
+	hyperapi "github.com/openshift/hypershift/api"
+	hyperv1 "github.com/openshift/hypershift/api/v1alpha1"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	cr "sigs.k8s.io/controller-runtime"
+	crclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type CreateNodePoolOptions struct {
+	Name        string
+	Namespace   string
+	ClusterName string
+	NodeCount   int32
+	Render      bool
+}
+
+func NewCreateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "nodepool",
+		Short: "Create a HyperShift NodePool",
+	}
+
+	opts := CreateNodePoolOptions{
+		Name:        "example",
+		Namespace:   "clusters",
+		ClusterName: "example",
+		NodeCount:   2,
+	}
+
+	cmd.Flags().StringVar(&opts.Name, "name", opts.Name, "The name of the NodePool")
+	cmd.Flags().StringVar(&opts.Namespace, "namespace", opts.Namespace, "The namespace in which to create the NodePool")
+	cmd.Flags().Int32Var(&opts.NodeCount, "node-count", opts.NodeCount, "The number of nodes to create in the NodePool")
+	cmd.Flags().StringVar(&opts.ClusterName, "cluster-name", opts.ClusterName, "The name of the HostedCluster nodes in this pool will join")
+
```

Comment:
how about we include params for:
- autoRepair
- autoscalingMin / autoscalingMax
- maxUnavailable / maxSurge
?

Reply:
Just read the PR desc. I'm good if you prefer to follow up in a different PR.